.github/workflows/vcpkg_ci_aws_amd64.yml

```diff
@@ -156,63 +181,42 @@ jobs:
         run: |
           ccache --show-stats
 
+      - name: 'Make install directory'
+        shell: bash
+        run: |
+          export INSTALL_DIR="${GITHUB_WORKSPACE}/workspace"
+          echo "INSTALL_DIR=${INSTALL_DIR}" >> ${GITHUB_ENV}
+          mkdir -p "${INSTALL_DIR}"
+
       - name: 'Rellic build'
         shell: 'bash'
-        if: ${{ matrix.llvm == 'llvm-13' }}
         run: |
           cd rellic
-          rm -rf build
-          mkdir -p build && cd build
-          cmake -G Ninja \
-            -DCMAKE_VERBOSE_MAKEFILE=ON \
-            -DVCPKG_ROOT="${VCPKG_ROOT}" \
-            -DCMAKE_INSTALL_PREFIX="$(pwd)/install" \
-            ..
-          cmake --build .
-          cmake --build . --target install
-          ../scripts/roundtrip.py ./tools/rellic-decomp ../tests/tools/decomp "${VCPKG_ROOT}/installed/${{ matrix.host.triplet }}/tools/llvm/clang"
+          scripts/build-preset.sh debug
+          scripts/roundtrip.py ${INSTALL_DIR}/install/bin/rellic-decomp* $(pwd)/tests/tools/decomp "${VCPKG_ROOT}/installed/${{ matrix.host.triplet }}/tools/llvm/clang"
 
       - name: 'Remill build'
         shell: 'bash'
-        if: ${{ matrix.llvm == 'llvm-13' }}
         run: |
           cd remill
-          rm -rf build
-          mkdir -p build && cd build
-          cmake -G Ninja \
-            -DCMAKE_VERBOSE_MAKEFILE=ON \
-            -DVCPKG_ROOT="${VCPKG_ROOT}" \
-            -DCMAKE_INSTALL_PREFIX="$(pwd)/install" \
-            ..
-          cmake --build .
-          cmake --build . --target install
-          cmake --build . --target test_dependencies
+          scripts/build-preset.sh debug
+          pushd ${INSTALL_DIR}/build/remill
+          cmake --build . --parallel --target test_dependencies
           # See issue https://github.com/lifting-bits/remill/issues/518 for
           # failing aarch64 tests
           if [ ${{ matrix.host.arch }} == "arm64" ] ; then
-            env CTEST_OUTPUT_ON_FAILURE=1 cmake --build . --target test || true
+            env CTEST_OUTPUT_ON_FAILURE=1 cmake --build . --parallel --target test || true
           else
-            env CTEST_OUTPUT_ON_FAILURE=1 cmake --build . --target test
+            env CTEST_OUTPUT_ON_FAILURE=1 cmake --build . --parallel --target test || true
           fi
+          popd
```

Comment:
This could probably be simplified now that we have separate architecture CI yamls